### DISCUSSION TITLE
How to delete termux and install latest fixed version

### DISCUSSION THREAD

**Discussion by jcubic** [termux/termux-app#3546] [Q&A] [answered]
I get a lot of errors from termux. I fix one got another one, right now I can't install anything or update because of the error

CANNOT LINK EXECUTABLE "/data/data/com.termux/files/usr/lib/apt/methods/https": library "libssl.so.1.1" not found: needed by /data/data/com.termux/files/usr/lib/libcurl.so in namespace (default)

I've installed openssl that I can't remove now because it ask me to remove important packages.

I just want install nodejs to test something on my phone, but there are errors. I can't update. Nothing is working.

Why can't I just delete termux and install it again, all I have when I did it is the same error state. 

How to delete termux and get updated version that works?

**Comment by agnostic-apollo** [comment, marked as answer]
Reinstall and then run `pkg upgrade`. Do not just update critical packages like `curl` without updating other packages.

**Reply by jcubic** [reply]
I'm not to able to upgrade got this error

```
Reading package lists... Done
E: The repository 'https://dl.bintray.com/grimler/game-packages-24 games Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
E: The repository 'https://dl.bintray.com/grimler/science-packages-24 science Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
```

**Reply by agnostic-apollo** [reply]
Install app from fdroid or github.

https://github.com/termux/termux-app#Installation

https://github.com/termux/termux-app#google-play-store-deprecated

**Reply by jcubic** [reply]
Thanks yes I just found play store is deprecated. Installed from fdroid, but now get this error after installing nodejs

```
CANNOT LINK EXECUTABLE "node": library "libcrypto.so.3" not found: needed by main executable
dpkg: error processing package nodejs (--configure):
 installed nodejs package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 nodejs
E: Sub-process /data/data/com.termux/files/usr/bin/dpkg returned an error code (1)
```

**Reply by agnostic-apollo** [reply]
Did you run `pkg upgrade` after installation?

**Reply by jcubic** [reply]
Ok solve it the fdroid also require upgrade
